app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -36,9 +62,13 @@ def destroy
       end
 
       def find_parent_config_template
```

Comment:
I see in your link that this filter's called, but both methods called `nested_obj`.  It seems to work here when I comment out the filter, what error do you see?